example/windows/test-app.bat

```diff
@@ -0,0 +1 @@
+node node_modules/react-native-test-app/windows/test-app.js "windows/" "Example"
```

Comment:
Instead of putting this `.bat` file here, we should add the script under `bin` in `react-native-test-app`'s `package.json`. See https://docs.npmjs.com/files/package.json#bin.

Something like:
```json
  "bin": {
    "install-windows-test-app": "windows/test-app.bat"
  }
```